tests/baselines/reference/2dArrays.types

```diff
@@ -35,17 +35,17 @@ class Board {
 >this.ships.every(function (val) { return val.isSunk; }) : boolean
 >                                                        : ^^^^^^^
 >this.ships.every : { <S extends Ship>(predicate: (value: Ship, index: number, array: Ship[]) => value is S, thisArg?: any): this is S[]; (predicate: (value: Ship, index: number, array: Ship[]) => unknown, thisArg?: any): boolean; }
->                 : ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+>                 : ^^^^^^^^^^^^^^^^^^^         ^^^     ^^^^^^^^     ^^^^^^^^^^     ^^^^^^^^^^^^^^^^^^^^^^^^^       ^^^^^^^^^^^^^^^^^^^^^^^         ^^^     ^^^^^^^^     ^^^^^^^^^^     ^^^^^^^^^^^^^^^^^^^^^^       ^^^^^^^^^^^^^^^^^^^

```

Comment:
These are great example - these parameters all come from `Array.prototype.every`, which is declared in the lib (obviously), so while we can copy the parameter name nodes to create this signature node, we *can't* copy the position on the nodes, since that would try and pull gibberish text from *this* file, if it didn't crash for violating some invariant somewhere.